.github/workflows/regression_test.yml

```diff
@@ -70,6 +70,12 @@ jobs:
             torch-spec: 'torch==2.5.1 --index-url https://download.pytorch.org/whl/cu121'
             gpu-arch-type: "cuda"
             gpu-arch-version: "12.1"
+          - name: CUDA Nightly
```

Comment:
why are these tests added?

Reply:
I followed https://github.com/pytorch/ao/pull/1283/files#diff-87efadde54d371bf3d7330fff24599a85d32c4982530dd4c7d7d7855e76489bbL28-L33 to update nightly. I have removed CUDA nightly and CPU nightly already.